src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -274,92 +310,103 @@ describe('visual-refresh-toolbar', () => {
-        await expect(page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)).resolves.toBeTruthy();
-        await page.hoverElement(splitPanelTriggerSelector);
-        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(1);
-        await page.keys(['Escape']);
-        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
-      })
-    );
+      test(
+        'Removes tooltip from split panel trigger on escape key press after showing from mouse hover',
+        setupTest({ theme, size, splitPanelPosition }, async page => {
+          await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
+          await expect(
+            page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)
+          ).resolves.toBeTruthy();
+          await page.hoverElement(splitPanelTriggerSelector);
+          await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(1);
+          await page.keys(['Escape']);
+          await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
+        })
+      );
 
-    //vr-toolbar-only
-    test(
-      'Shows tooltip correctly for split panel trigger for keyboard (tab) interactions',
-      setupTest({ theme, size }, async page => {
-        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
-        await expect(page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)).resolves.toBeTruthy();
-        await page.click(splitPanelTriggerSelector);
-        await expect(page.isFocused(splitPanelTriggerSelector)).resolves.toBeTruthy();
-        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
-        await page.click(splitPanelTriggerSelector);
-        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
-        //navigate away from slide panel
-        await page.keys(['Shift', 'Tab', 'Null']); // to last breadcrumb - be aware if breadcrumb has tooltip
-        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
-        //navigate back to drawer trigger
-        await page.keys(['Tab']);
-        await expect(page.isFocused(splitPanelTriggerSelector)).resolves.toBeTruthy();
-        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(1);
-        await page.keys(['Shift', 'Tab', 'Null']);
-        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
-      })
-    );
+      test(
+        'Shows tooltip correctly for split panel trigger for keyboard (tab) interactions',
+        setupTest({ theme, size, splitPanelPosition }, async page => {
+          await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
+          await expect(
+            page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)
```

Comment:
I saw this trigger multiple times in these test suites. Let's create a selector for them

Reply:
this would involve altering the classes, moving the creation of some classes into the test-classes scss file and then using that in the test-util.  It will be a small refactor but best to be in its own PR